src/pages/docs/packaging-applications/package-repositories/artifactory-generic-feeds.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: src/layouts/Default.astro
+pubDate: 2023-11-20
+modDate: 2023-11-20
+title: Artifactory Generic Feeds
+description: Configuring an Artifactory Generic Feed in Octopus
+navOrder: 60
+---
+
+:::div{.warning}
+From version **Octopus 2024.1** we support Artifactory Generic Repositories. This functionality is behind the `ArtifactoryGenericFeedFeatureToggle` feature toggle, to request this functionality early, please context [support](https://octopus.com/support).
+:::
+
+:::div{.warning}
+To use the Artifactory Generic Feeds feature you will need a PRO or higher license of Artifactory
+:::
+
+If you're using an Artifactory Generic Repository, you can create a feed in Octopus and use artifacts as part of your deployments. To create a feed go to **Library ➜ External feeds**. Select the **Add feed** button and selecting the _Artifactory Generic Repository_ feed type.
+
+You will then need to provide a feed name, the Artifactory repository name, an [access token](https://oc.to/ArtifactoryAccessToken) and the repository [Artifact Path regex](https://oc.to/ArtifactoryGenericLayouts).
+
+:::figure
+![](/docs/packaging-applications/package-repositories/images/artifactory-generic-feed-creation.png)
+:::
+
+Artifactory generic feeds accommodate files of any type without requiring a specific file name structure. To handle this Artifactory supports [custom layouts](https://oc.to/ArtifactoryGenericLayouts). Custom layouts produce a regex expression that the file path and name must match, enabling Artifactory to extract the file type, version and module. This customizable package structure does not match Octopus's own expected [package versioning conventions](/docs/packaging-applications/create-packages/versioning) used in other feeds. To handle this we depend on the Artifact Path Pattern regex expression available from Artifactory to be set on the feed in Octopus. To find the _Artifact Path Pattern_ go to **Administration ➜ Layouts ➜ Regular Expression View ➜ Resolve** and copy the Artifact Path Regex expression. The default layout regex for a new repository and layout is <code>(?<orgPath>.+?)/(?<module>[^/]+)/(?<module>\2)-(?<baseRev>[^/]+?)\.(?<ext>(?:(?!\d))[^\-/]+|7z)</code>.
```

Comment:
Perhaps we could mention that the default value here is the one from the `simple-layout`